### ISSUE TITLE
Error when viewing peers in the downloads tab

### ISSUE THREAD

**Issue by loadletter** [Tribler/tribler#2935]
- [X] I have read CONTRIBUTING.rst

- [X] I have tried with the latest pre-release version and I still can reproduce the issue.

##### Tribler version/branch+revision: #####
7.0.0-RC1
##### Operating system and version: #####
Debian 8 (Jessie)
##### Expected behavior: #####
Selecting a download and then clicking on peers tab the GUI should display details about peers
##### Actual behavior: #####
After a few seconds a popup comes up
```
Request error
An error occurred during the request:

'big_number' object has no attribute 'to_string'
```
And then repeats blacking out the entire interface

Maybe it's just libtorrent being too old? (Version: 0.16.18-1)

**Comment by qstokkink** [comment]
Can confirm that `0.16.18` does not have `to_string`:
https://github.com/oerdnj/libtorrent-rasterbar/blob/37030c58158fb9a3d92a021241e295fda59958d6/bindings/python/src/big_number.cpp
But 1.10 does:
https://github.com/oerdnj/libtorrent-rasterbar/blob/master/bindings/python/src/big_number.cpp

**Comment by devos50** [comment]
Fixed by #2936
